autoload/gitgutter/hunk.vim

```diff
@@ -55,15 +55,19 @@ function! gitgutter#hunk#next_hunk(count) abort
   endif
 
   let current_line = line('.')
+  let hunk_total = len(hunks)
+  let hunk_number = 1
```

Comment:
I don't think you need a new variable.  You can use `hunk_count` and add 1 when you echo the message.

Reply:
I don't think so. When you type `3]c`, the value of `hunk_count` should be `3`, regardless of the hunk position.

Reply:
I don't understand.  `3]c` jumps forward 3 hunks from wherever the cursor is.

Reply:
Suppose that you have 11 hunks and you are on the fourth hunk, i.e. `Hunk 4 of 11`. `3]c` jumps forward 3 hunks as you say, but the message we want to see is `Hunk 7 of 11` instead of `Hunk 3 of 11`.

Reply:
Oh, I see.  Thanks :)

Reply:
I don't like that `hunk_total` and `hunk_number` are unnecessarily declared even for users who turn `g:gitgutter_show_msg_on_hunk_jump` option off, but I don't think it affects performance so much.

Reply:
I could simply add

```vim
if g:gitgutter_show_msg_on_hunk_jumping
  redraw | echo printf('Hunk %d of %d', index(hunks, hunk) + 1, len(hunk))
endif
```

without adding those variables, but I don't think this is  clever either.
Do you have any other good ideas? 🤔

Reply:
Actually I quite like that ;)

Reply:
Oh, okay. Then I replace it with the code above ;)